packages/shared/src/server/repositories/clickhouse.ts

```diff
@@ -51,6 +47,13 @@ export async function upsertClickhouse<
       );
       await Promise.all(
         opts.records.map((record) => {
+          // drop trailing s and pretend it's always a create.
```

Comment:
why do we need this change?

Reply:
@maxdeichmann I want to derive a valid event to store in S3 based on the ClickHouse table. For traces and scores that's easy as they become `trace-create` and `score-create`. For observations we would want `span-create`, `generation-create`, and `event-create` though. Hence, I had to become a bit more creative here.